CMakeLists.txt

```diff
@@ -2,6 +2,9 @@
 cmake_minimum_required(VERSION 3.18)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+# Set fetched URL contents to the time of extraction, not the timestamps in the archive.
+cmake_policy(SET CMP0135 NEW)
```

Comment:
Does this cause an error if using an older cmake that doesn't have this policy? Do we have to wrap it around a check?